sdk/wren-pydantic/README.md

```
@@ -30,6 +30,25 @@ Runnable demos:
 - [`examples/pydantic_ai_structured_demo.py`](./examples/pydantic_ai_structured_demo.py) —
   same shape with `output_type=` for structured / validated agent output.
 
+### Routing through OrcaRouter
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)